content/applications/inventory_and_mrp/quality/quality_management/quality_alerts.rst

```diff
@@ -1,81 +1,89 @@
-- :guilabel:`Responsible`: the individual responsible for managing the quality alert
-- :guilabel:`Tags`: classify the quality alert based on user-created tags
-- :guilabel:`Root Cause`: the cause of the quality issue, if known
-- :guilabel:`Priority`: assign a priority between one and three stars to ensure more
-  urgent issues are prioritized
-
-The tabs at the bottom of the form can be used to provide additional information to quality teams:
-
-- :guilabel:`Description`: provide additional details about the quality issue
-- :guilabel:`Corrective Actions`: the method for fixing affected products
-- :guilabel:`Preventive Actions`: procedures for preventing the issue from occurring in
-  the future
-- :guilabel:`Miscellaneous`: the product vendor (if applicable), the company that produces
-  the product, and the date assigned
-
-.. image:: quality_alerts/quality-alert-form.png
+.. |MO| replace:: :abbr:`MO (Manufacturing Order)`
+
+In Odoo *Quality*, *quality alerts* are used to notify quality teams of product defects or other
+issues. Quality alerts can be created from a manufacturing or inventory order, from a work order in
+the *Shop Floor* module, or directly within the *Quality* app.
+
+Create a quality alert
+======================
+
+There are four ways to create a new quality alert:
+
+- Navigate to :menuselection:`Quality -->  Quality Control --> Quality Alerts`, and then click
+  :guilabel:`New` to open a quality alert form.
+- Navigate to :menuselection:`Manufacturing --> Operations --> Manufacturing Orders`, and then
+  select an |MO|. Click the :guilabel:`Quality Alert` button at the top of the |MO| to open a
+  quality alert form in a new page.
+- Open the :menuselection:`Inventory` app, click the :guilabel:`# To Process` button on an inventory
+  order type card (Receipts, Delivery Orders, etc.), and then select an order. Click the
+  :guilabel:`Quality Alert` button at the top of the order to open a quality alert form in a new
+  page.
+- Open the :menuselection:`Shop Floor` module, and then select a work center from the navigation bar
+  at the top of the page. Then, click the :guilabel:`⋮ (three vertical dots)` button at the
+  bottom-right of a work order card to open the :guilabel:`What do you want to do?` menu. Select the
+  :guilabel:`Create a Quality Alert` option from this menu to open a quality alert in a pop-up
+  window.
+
+.. note::
+   Depending on how a new quality alert form is opened, certain fields on the form may already be
+   filled in. For example, if a quality alert is created from a work order card in the *Shop Floor*
+   module, the :guilabel:`Product` and :guilabel:`Work Center` fields will be pre-filled.
+
+After opening a new quality alert form, fill it out as follows:
+
+- Enter a short :guilabel:`Title` that summarizes the issue with the product.
+- If the product issue was discovered at a specific work center, select it from the :guilabel:`Work
+  Center` drop-down menu.
```

Comment:
```suggestion
- Enter a short :guilabel:`Title` that summarizes the issue with the product.
- If it not already auto-populated, fill in the :guilabel:`Product`, :guilabel:`Product Variant`,
  and :guilabel:`Lot` accordingly, if necessary.
- If the product issue was discovered at a specific work center, select it from the :guilabel:`Work
  Center` drop-down menu.
```

Reply:
just added a quick mention about those fields for a more "complete" overview of the form

Reply:
These aren't the only fields that can auto-populate. I think it's simpler to just have the note.

Reply:
For sure. I totally understand that. However, in your note about some fields auto-populating, you mention the :guilabel:`Work Center' field, and then proceed to explain its use below. That's why I think it may be best to just include all the fields on the Quality Alert form for a full overview. I completely get where you're coming from, but for the sake of docs, I feel like it's always best to explain everything on the screen.